sysutils/rename/Portfile

```
@@ -0,0 +1,30 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        ap rename 0da9faa241093c90a40b15a9ff4b36e9858d4516
+name                rename
+version             1.601
+revision            0
+categories          sysutils
+license             GPL-1+
+maintainers         @dylanarmstrong
+
+description         rename files using regexp passed from command line
+long_description    This program renames files according to modification \
+                    rules specified on the command line. If no filenames \
+                    are given on the command line, a list of filenames \
+                    will be expected on standard input.
+
+homepage            http://plasmasturm.org/code/rename
+
+checksums           rmd160  e7b89cf872310d7a97134df0d182dbd52ed01360 \
+                    sha256  a146b7968d8ecf39e559cf76b62270c961ba153a8ab46092cf22ace9ef3905fa \
+                    size    14907
+
```

Ports should not be implicitly depending on system binaries.  This port should assert an explicit dependency on Perl:

```suggestion

depends_lib-append  path:bin/perl:perl5
```

Using the `path:` style specification allows us to say that this port is fine as long as _some_ Perl is present, but if indeed there is none installed, then install Perl 5.